kuma/javascript/src/index.jsx

```diff
@@ -33,88 +34,104 @@ if (container) {
             // navigation between pages. Currently the single page app
             // handles document pages and search results
             app = (
-                /* StrictMode is a tool for highlighting potential problems
-                   in an application. Like Fragment, StrictMode does not
-                   render any visible UI. It activates additional checks and
-                   warnings for its descendants. 
-                   @see https://reactjs.org/docs/strict-mode.html */
-                <React.StrictMode>
-                    <SinglePageApp
-                        initialURL={data.url}
-                        initialData={data.documentData}
-                    />
-                </React.StrictMode>
+                <SinglePageApp
+                    initialURL={data.url}
+                    initialData={data.documentData}
+                />
             );
+            // app = (
```

Comment:
I like what you've done with moving StrictMode-wrapping downstairs, but I'd vote for also deleting the commented out code.

Reply:
This is one of the many reasons why code reviews are good.